### DISCUSSION TITLE
Support custom domains for shared album links

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#541] [Enhancements]
Instead of `albums.ente.io/xyz`, I should be able to use `photos.mydomain.com/xyz`.

**Comment by VirtualPanther** [comment]
I would expand on that and would love to see my own domain pointing to my ente site. This is how I use Smugmug. My friends and colleagues just go to my personal website and are greeted with a custom homepage, asking for a visitor password. Love your app and the project itself. But I literally see no ability to send anyone to visit my site, as they will not remember the ente name and even if they did, there is no "visitor" account with restrictions.

**Comment by sorbanbela** [comment]
It is a really good idea! I would rather subscribe to a paid plan and use my domain instead of self hosting.

**Comment by ua741** [comment]
https://ente.io/help/photos/features/sharing-and-collaboration/custom-domains/#custom-domains
